content/hacking-atom/sections/hacking-on-atom-core.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Hacking on Atom Core
+---
+### Hacking on Atom Core
```

Comment:
We may want a blurb at the top like the following:

> If you're hitting a bug in Atom or just want to experiment with adding a feature to the core of the system, you'll want to run Atom in dev mode with access to a local copy of the Atom source.

Then we could have links to instructions for cloning and building Atom and a separate section for running Atom in dev mode with the `ATOM_DEV_RESOURCE_PATH` variable assign.